pkg/apis/kops/validation/instancegroup.go

```diff
@@ -133,6 +133,12 @@ func ValidateInstanceGroup(g *kops.InstanceGroup) error {
 		return err
 	}
 
+	if g.Spec.RollingUpdate != nil {
+		if errs := validateRollingUpdate(g.Spec.RollingUpdate, field.NewPath("rollingUpdate")); len(errs) > 0 {
+			return errs.ToAggregate()
```

Comment:
I'm curious as to why `ValidateInstanceGroup()` and `ValidateCluster()` return on the first found set of errors rather than gather and return all errors. Would this be something to fix?

Reply:
Probably yes - apimachinery introduced the idea of gathering all the errors, and we had something like validation that didn't use the machinery previously.  The apimachinery approach is better.  So yes, it would be great to gather and return all the errors (and print them all also, if we're not already doing so!)

Reply:
I'll take that up in a separate PR.